newIDE/app/src/UI/SelectOption.js

```diff
@@ -10,6 +10,7 @@ type Props = {|
   value: string | number | boolean,
   primaryText: MessageDescriptor,
   disabled?: boolean,
+  areUserDefinedValues?: boolean,
```

Comment:
This works.
We've also done something different in `MarkdownText.js`
2 different props:
- source & translatableSource

Check it out, maybe we should copy something similar with `translatablePrimaryText` and `primaryText` ?